src/store/counterData/reducer.js

```diff
@@ -0,0 +1,39 @@
+const initialState = {
+  counter: 0,
+  toDo: [],
+};
+
+function rootReducer(state = initialState, action) {
+  switch (action.type) {
+    case 'INCREASE_COUNTER':
+      return {
+        ...state,
+        counter: action.payload,
+      };
+    case 'DECREASE_COUNTER':
+      return {
+        ...state,
+        counter: action.payload,
+      };
+    case 'ADD_TODO':
+      return {
+        ...state,
+        toDo: [...state.toDo, action.payload],
```

Comment:
if you add the same item which already exist? it will add it, please make it impossible to add the same items.